core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3685,6 +3686,23 @@ class KafkaApis(val requestChannel: RequestChannel,
 
     requestHelper.sendResponseExemptThrottle(request, moveControllerResponse)
   }
+
+  def handleXinfraTopicCreateRequest(request: RequestChannel.Request): Unit = {
```

Comment:
Talked offline.  Instead of "creating", this API sounds more like "mark federated topic"